File: IOS/demodice/demodice/ViewController.m
//
//  ViewController.m
//  demodice
//
//  Created by Tyson Costello on 4/15/14.
//  Copyright (c) 2014 codeWithChris. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
    self.model = [[DiceDataController alloc] init];
    
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
- (IBAction)rollButtonClicked:(id)sender {
    int leftRoll = [self.model GetDiceRoll];
    int rightRoll = [self.model GetDiceRoll];
    
    [self.leftDie showDie:leftRoll];
    [self.rightDie showDie:rightRoll];
    
    NSString *sumText = [NSString stringWithFormat:@"Sum is %d", leftRoll + rightRoll];
    
    self.sumLabel.text = sumText;
    
}

@end

File: SwiftTutorial/Example App/Example App/ViewController.swift
//
//  ViewController.swift
//  Example App
//
//  Created by Tyson Costello on 1/20/15.
//  Copyright (c) 2015 com.Digitys. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet var myLabel: UILabel!
    
    @IBAction func buttonPressed(sender: AnyObject) {
        
        myLabel.text = "It worked"
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        println("Hello World!")
        //myLabel.text = "It worked!"
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

